components/Accordion/Accordion.jsx

```diff
@@ -222,15 +271,20 @@ Accordion.defaultProps = {
 };
 
 Accordion.propTypes = {
-  /** Edit only for themification */
+  /** Theme natively passed by Quantum theme. Don't change this prop */
   theme: PropTypes.shape({
     colors: PropTypes.object,
     spacing: PropTypes.object,
     baseFontSize: PropTypes.number,
   }),
   /** Set true to keep only one item open and close the remainings on click */
   keepOnlyOneOpen: PropTypes.bool,
-  /** Items to render in accordion */
+  /**
```

Comment:
Why?

Reply:
this comments in jsx are used in quantum-storytook-ui to render a explanation of the props use